src/plugins/mf_priority.cpp

```diff
@@ -152,6 +152,7 @@ static void add_special_association (flux_plugin_t *p, flux_t *h, int userid)
     a->cur_active_jobs = 0;
     a->active = 1;
     a->held_jobs = std::vector<long int>();
+    a->max_nodes = 2147483647;
```

Comment:
You're going for the maximum integer value here? I think it would be a bit cleaner to `#include <stdint.h>` and use `INT32_MAX` or `<limits.h>` and `INT_MAX`. Or maybe pick a 'nicer' number like 10000000? I was kind of surprised to see this number here seemingly out of nowhere. An inline comment might be helpful.

Reply:
Oh that's good to know, and a good suggestion - thanks! I think because this is C++, I should use `#include <cstdint>` here. And perhaps I should change `max_nodes` here to be of type `std::int32_t`. Any opinions on this choice?

Reply:
That sounds fine to me, since it looks like `int` is guaranteed to be at least 16 bits but can be more, so `INT_MAX` might vary across systems and you need a stable number for testing. You could also keep it as an `int` and use `INT16_MAX`, up to you.

Reply:
I think I'll just use `INT16_MAX`; thanks!